benchexec/result.py

```diff
@@ -55,6 +57,8 @@
 _PROP_LABEL =        'unreach-label'
 # currently used by SV-COMP (http://sv-comp.sosy-lab.org/2016/rules.php):
 _PROP_CALL =         'unreach-call'
+# extends _PROP_CALL for any error function:
+_PROP_CALL_SPECIFIC ='unreach-call,'
```

Comment:
Currently property name should be equal to _PROP_* constants, thus for property:
`LTL(G ! call(__VERIFIER_error()))`
property name is `_PROP_CALL`.
If we extend this property to any error function:
`LTL(G ! call(<error_function>()))`,
then information about <error_function> is missing in `_PROP_CALL` name.
Can we extend old format to support always property with any error function? 
In this case by default property name will be the following:
`_PROP_CALL(__VERIFIER_error)`
Also only one constant `_PROP_CALL` will be required.